great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -184,6 +183,76 @@ def build_parameters(
                 parameter_values=parameter_values,
             )
 
+    def resolve_evaluation_dependencies(
+        self,
```

Comment:
I don't see this method using `self`.
Which leads me to believe this should be either a `class` or `static` method.

Reply:
@Kilo59 Please see line 202.  Thanks.

Reply:
Ah, I missed it.